client/src/App.js

```diff
@@ -1,9 +1,43 @@
 import "./App.css";
+import { Switch, Route, Redirect, useLocation } from "react-router-dom";
+import Navigation from "./components/Navigation";
+import StudentsPage from "./components/StudentsPage";
+import CoursesPage from "./components/CoursesPage";
 
 function App() {
+  const location = useLocation();
+
+  function renderHeaderText() {
+    let headerText;
+    if (location.pathname.includes("courses")) {
+      headerText = "Courses";
+    } else if (location.pathname.includes("students")) {
+      headerText = "Students";
+    }
```

Comment:
I like this, but maybe in the future you just want to pass a prop to set the title, or maybe create component and move this logic there

Reply:
Good catch! I'll see if this component gets longer, then I'll definitely  take a look at it!